war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -325,7 +351,11 @@ public void addUserToOrganization(final String organizationId, final String user
     *       When user already exists in project
     */
    public void addUserToOrganization(final String organizationId, final String userName) throws UserAlreadyExistsException {
-      addUserToOrganization(organizationId, userName, readDefaultRoles(organizationId));
+      List<String> roles = readDefaultRoles(organizationId);
+      if (roles == null) {
+         roles = Collections.emptyList();
```

Comment:
When there are no default roles set, that attribute value does not exist in organization metadata. And if we then add new user, this null is read as default roles and is added to that new user. And then an exception is thrown when reading user and its roles because empty list is expected, not null.